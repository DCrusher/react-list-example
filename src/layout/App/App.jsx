import React from 'react';

import styles from './App.sss';

const App = () => (
    <div className={styles.base} >
        <a href="#">App</a>
    </div>
);

export default App;
