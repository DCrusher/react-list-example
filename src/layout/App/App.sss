@import 'assets/css/constants.sss'

.base
  background-color: $blue

a
  color: #fff
