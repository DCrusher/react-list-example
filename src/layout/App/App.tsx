import * as React from 'react';

import Counter from 'components/counter/Counter';

import * as styles from './App.sss';

const App: React.SFC = () => (
    <div
        className={styles.base}
    >
        <a href="#">App</a>
        <Counter />
    </div>
);

export default App;
