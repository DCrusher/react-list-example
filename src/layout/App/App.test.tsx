import * as React from 'react';
import { shallow }  from 'enzyme';
import enzymeToJson from 'enzyme-to-json';
import App from './App';

test('snapshot matching', () => {
    const component = shallow(
        <App />
    );

    expect(enzymeToJson(component)).toMatchSnapshot();
});