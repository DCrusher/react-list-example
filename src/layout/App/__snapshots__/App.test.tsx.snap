// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot matching 1`] = `
<div
  className="base"
>
  <a
    href="#"
  >
    App
  </a>
  <Counter />
</div>
`;
